python/tests/utils/test_to_string.py

```diff
@@ -0,0 +1,64 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+# @file
+# @author Neil Vaytet
+
+import numpy as np
+import scipp as sc
+import scipp.utils as su
```

Comment:
I am not sure we should introduce a new submodule into scipp without more curation/triaging of what goes in. Is there a way to keep this more private and not part of scipp?

Reply:
yes i can do that

Reply:
it seems if i don't want these low-level functions to be part of a submodule, I may have to introduce some `sys.path.append(...)` stuff in here.
Is that ok or should I look for another solution?
e.g.
```Py
import sys
sys.path.append('../src/scipp/utils')
```

Reply:
Maybe just make it `scipp._util`, following the "convention" for marking things as private?

Reply:
See also https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces.